packages/optimizer/spec/transformers/valid/ServerSideRendering/converts_media_attribute_to_css/expected_output.html

```diff
@@ -0,0 +1,28 @@
+<html ⚡ i-amphtml-layout i-amphtml-no-boilerplate>
+<head><style amp-runtime></style>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <script async custom-element="amp-img" src="https://cdn.ampproject.org/v0/amp-img-0.1.js"></script>
+
+
```

Comment:
Removal of the boilerplate shouldn't leave empty lines behind.

Reply:
that's going to be removed by a different transformer (not used in the test)

Reply:
This will be fixed by a different transformer